reciprocalspaceship/algorithms/scale_merged_intensities.py

```diff
@@ -245,49 +291,35 @@ def scale_merged_intensities(ds, intensity_key, sigma_key, output_columns=None,
     if 'CENTRIC' not in ds:
         ds.label_centrics(inplace=True)
 
-    if output_columns:
-        outputI, outputSigI, outputF, outputSigF = output_columns
-    else:
-        columns = ["FW-I", "FW-SIGI", "FW-F", "FW-SIGF"]
-        outputI, outputSigI, outputF, outputSigF = columns
-        
+    if output_columns is None:
+        output_columns = ["FW-I", "FW-SIGI", "FW-F", "FW-SIGF"]
+    outputI, outputSigI, outputF, outputSigF = output_columns
+
+    multiplicity = compute_structurefactor_multiplicity(ds.get_hkls(), ds.spacegroup)
```

Comment:
`multiplicity = ds.compute_multiplicity()["EPSILON"].to_numpy()`

ought to work